vendor/reentrant_mutex.rb

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2014 Boris Bera
```

Comment:
Can you document where this is vendored from?

Reply:
You got it!